eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -87,7 +93,27 @@ def callback_full(theta, lc, model, prior1, prior2, priortype, freenames):
 
     # Get the best fit params
     fit_params = results.x
-
+    
+    # Create table of results
+    t_results = table.Table([freenames, fit_params], 
+                            names=("Parameter", "Mean")  ) 
+
+    # Save transmission spectrum
+    # indices of fitted rps for all channels
+    # ind_spec = np.array([i for i in range(len(freenames)) if 'rp' in freenames[i]])
+    # # get wavelengths at middle of bin
+    # wave_low = meta.wave_low[lc.fitted_channels]
+    # wave_hi = meta.wave_hi[lc.fitted_channels]
+    # wave_mid = (wave_hi+wave_low)/2.
+    # # Save transmission spectrum
+    # # indices of fitted rps for all channels
+    # ind_spec = np.array([i for i in range(len(freenames)) if 'rp' in freenames[i]])
+    # if len(ind_spec):
+    #     t_spec = table.Table([wave_low, wave_mid, wave_hi, fit_params[ind_spec]**2.*1e6, \
+    #                       np.zeros_like(wave_low), np.zeros_like(wave_low)],
+    #                      names=("wave_low_um", "wave_mid_um", "wave_upp_um", "RpRs2_ppm", "err_low_ppm", "err_upp_ppm"))
+    # else:
+        # t_spec = None
```

Comment:
I would delete all these commented out chunks of code unless @kevin218 disagrees. Having large commented out blocks of code makes things hard to read through in the future. If ever we need the code, it is already saved in the Git history

Reply:
I'll do it if Kevin thinks that's the best way to go, I left them there because I thought it was my only instruction :)

Reply:
done